test/unit/BondingManager.js

```diff
@@ -123,46 +123,49 @@ contract("BondingManager", accounts => {
         it("should fail if current round is not initialized", async () => {
             await fixture.roundsManager.setMockBool(functionSig("currentRoundInitialized()"), false)
 
-            await expectThrow(bondingManager.transcoder(5, 10, 1))
+            await expectThrow(bondingManager.transcoder(5, 10))
+        })
+
+        it("should fail if the current round is locked", async () => {
+            await fixture.roundsManager.setMockBool(functionSig("currentRoundLocked()"), true)
+
+            await expectThrow(bondingManager.transcoder(5, 10))
```

Comment:
Since we added a reason string to the require statement for checking if the current round is locked, can we use the [expectRevertWithReason()](https://github.com/livepeer/protocol/blob/pm/test/helpers/expectFail.js) helper to make sure the tx is failing for the correct reason?

Reply:
fixed in d144143